api/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
-
+from django.utils import timezone
 # Create your models here.
+
+
+class Projects(models.Model):
+	PROJECT_STATUS = (
+		('O', 'Opened'),
```

Comment:
'open' is a much better short name than 'o', since your code will look like:

`if project.project_status == 'open': ..`

instead of of:

`if project.project_status == 'o': ..`

(explicit is always better than implicit)